src/follower.py

```diff
@@ -73,35 +73,60 @@ def stop(self):
 
     """
     Turn the wheels towards the last recognized object.
+    Pans the camera towards the last recognized object.
     """
     def turn(self):
+        self.pan_camera()
         self.convert_camera_angle()
         self._fw.turn(self._turn_angle)
 
 
     """
-    Converts the camera's angle scale to the same scale as the wheels.
+    Follows the tag by panning camera towards the same direction as the wheels.
+    
+    If the direction of the object is more/less than the distance then set the camera 3 steps 
```

Comment:
Few problems:

1. Why does this have anything to do with distance? It exclusively deals with angles.
2. Don't say 3 steps, say how many degrees. Steps could be anything. This was really confusing when we first found out SunFounder was using steps, so imagine how confused people reading our code would be. Our code should be everything that we wanted their code to be.
3. Why does `pan_camera` compare the turn angle to the distance from the vehicle? How do you compare a distance in meters to an angle in degrees?
4. `self._distance` is the **z** distance (the distance TO the vehicle). It looks like you wanted the x distance.